README.md

```diff
@@ -47,16 +47,20 @@ data. You can run the checks locally with the `check` command:
 cargo run check
 ```
 
+Note that, if you're using this for the first time, some of these checks will be skipped due to missing API tokens.
```

Comment:
I'm not sure it makes sense to qualify this with "first time". After all, if they run again (or a hundred more times), the tool will still skip some checks. 

```suggestion
Note that some of these checks will be skipped due to missing API tokens.
```